### PR TITLE
Add GluonNLP

### PR DESCRIPTION
**Pull request by leezu**
Adds GluonNLP 0.9 pre-release version. Version 0.9 will be released once MXNet 1.6 is released.
As DLC relies on MXNet pre-release version, we need to use GluonNLP pre-release version for compatibility.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by leezu** [comment]
@saimidu please advise about the CI error

``` python
Error initializing plugin EntryPoint(name='Windows (alt)', value='keyrings.alt.Windows', group='keyring.backends').
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/keyring/backend.py", line 203, in _load_plugins
    init_func = ep.load()
  File "/usr/local/lib/python3.6/dist-packages/importlib_metadata/__init__.py", line 94, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/lib/python3/dist-packages/keyrings/alt/Windows.py", line 9, in <module>
    from . import file_base
  File "/usr/lib/python3/dist-packages/keyrings/alt/file_base.py", line 13, in <module>
    from keyring.util.escape import escape as escape_for_ini
ModuleNotFoundError: No module named 'keyring.util.escape'
```

**Comment by leezu** [comment]
Blocked by the use of `pip>=20`: 

```
[91mDEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
[0m[91mERROR: aws_mxnet_mkl-1.6.0rc0-py2.py3-none-manylinux1_x86_64.whl is not a supported wheel on this platform.
```

File: docker/1.6.0/py2/Dockerfile.cpu [hunk 1]
```diff
@@ -33,7 +33,7 @@ RUN apt-get update \
  && ln -s $(which python) /usr/local/bin/python
 
 RUN pip --no-cache-dir install --upgrade \
-    pip \
+    pip==19.3.1 \
```
Comment (arjkesh) [inline docker/1.6.0/py2/Dockerfile.cpu]: thanks for adding these, and unpinning awscli elsewhere

**Comment by saimidu** [comment]
We will merge this PR when other reviews are complete.

**Comment by leezu** [comment]
@arjkesh I think it's prudent to leave it pinned as upstream undergoes heavy development to modernize their codebase, making the risk of future bugs higher than in the past. I just wanted to point out in the comment that the earlier breakage is not related to the Python 2 support.

**Pull request merged**